src/content/glossary/index.md

```diff
@@ -265,11 +272,11 @@ Short for "decentralized finance," a broad category of [dapps](#dapp) aiming to
 
 ### difficulty {#difficulty}
 
-A network-wide setting that controls how much computation is required to produce a [proof-of-work](#pow).
+A network-wide setting that controls how much computation is required to produce a valid nonce in [proof-of-work](#pow) networks.
 
 ### difficulty bomb {#difficulty-bomb}
 
-Planned exponential increase in [proof-of-work](#pow) [difficulty](#difficulty) setting designed to motivate the transition to [proof-of-stake](#pos), reducing the chances of a [fork](#hard-fork)
+Planned exponential increase in [proof-of-work](#pow) [difficulty](#difficulty) setting that was designed to motivate the transition to [proof-of-stake](#pos), reducing the chances of a [fork](#hard-fork). This was only a feature of Ethereum before it transitioned to proof-of-stake.
```

Comment:
Wording feels kinda awkward here.

```suggestion
Planned exponential increase in [proof-of-work](#pow) [difficulty](#difficulty) setting that was designed to motivate the transition to [proof-of-stake](#pos), reducing the chances of a [fork](#hard-fork). The difficulty bomb was deprecated at [The Merge](/upgrades/merge/).
```

Or maybe 'remove' is perhaps more accurate?

```suggestion
Planned exponential increase in [proof-of-work](#pow) [difficulty](#difficulty) setting that was designed to motivate the transition to [proof-of-stake](#pos), reducing the chances of a [fork](#hard-fork). The difficulty bomb was removed at [The Merge](/upgrades/merge/).
```